data/provider/src/main/java/tech/pegasys/artemis/api/schema/BeaconBlockBody.java

```diff
@@ -69,4 +70,32 @@ public BeaconBlockBody(tech.pegasys.artemis.datastructures.blocks.BeaconBlockBod
             .map(SignedVoluntaryExit::new)
             .collect(Collectors.toList());
   }
+
+  public tech.pegasys.artemis.datastructures.blocks.BeaconBlockBody asInternalBeaconBlockBody() {
+    return new tech.pegasys.artemis.datastructures.blocks.BeaconBlockBody(
+        randao_reveal.asInternalBLSSignature(),
+        new tech.pegasys.artemis.datastructures.blocks.Eth1Data(
+            eth1_data.deposit_root, eth1_data.deposit_count, eth1_data.block_hash),
+        graffiti,
+        SSZList.createMutable(
+            proposer_slashings.stream().map(ProposerSlashing::asInternalProposerSlashing),
+            proposer_slashings.size(),
```

Comment:
If we decide to use constants, blocks/BeaconBlockBody has all the constants in line comments with each list.